src/FamousFigure.java

```diff
@@ -0,0 +1,188 @@
+import components.map.Map;
+import components.map.Map2;
+import components.simplewriter.SimpleWriter;
+import components.simplewriter.SimpleWriter1L;
+
+/**
+ * Holds information about the artist Raphael.
+ *
+ * @author Nicole Young
+ */
+public final class FamousFigure {
+
+    /**
+     * Private constructor so this utility class cannot be instantiated.
+     */
+    private FamousFigure() {
+    }
+
+    /**
+     * A work done by the artist.
+     *
+     * @param figureInfo
+     *            the map containing information about the figure
+     * @return a randomly chosen work by the figure
+     */
+    public static String work(Map<String, String> figureInfo) {
```

Comment:
This is great for a proof of concept. Keep in mind that you will need to rewrite these methods to be instance methods. For example, I would assume the map here is your implementation details.

Reply:
By the way, a map is a great data structure for this!